zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/DecisionRequirementsHandler.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.handlers;
+
+import static io.camunda.exporter.utils.ExporterUtil.tenantOrDefault;
+
+import io.camunda.exporter.store.BatchRequest;
+import io.camunda.webapps.schema.entities.operate.dmn.definition.DecisionRequirementsEntity;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.ValueType;
+import io.camunda.zeebe.protocol.record.intent.DecisionRequirementsIntent;
+import io.camunda.zeebe.protocol.record.value.deployment.DecisionRequirementsRecordValue;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+public class DecisionRequirementsHandler
+    implements ExportHandler<DecisionRequirementsEntity, DecisionRequirementsRecordValue> {
+  private static final Charset CHARSET = StandardCharsets.UTF_8;
+
+  private final String indexName;
+
+  public DecisionRequirementsHandler(final String indexName) {
+    this.indexName = indexName;
+  }
+
+  @Override
+  public ValueType getHandledValueType() {
+    return ValueType.DECISION_REQUIREMENTS;
+  }
+
+  @Override
+  public Class<DecisionRequirementsEntity> getEntityType() {
+    return DecisionRequirementsEntity.class;
+  }
+
+  @Override
+  public boolean handlesRecord(final Record<DecisionRequirementsRecordValue> record) {
+    return record.getIntent().equals(DecisionRequirementsIntent.CREATED);
```

Comment:
PS: Notice that in https://github.com/camunda/camunda/blob/6eda1cf3a6eee69af235ce44c84d4c84de75a23c/zeebe/exporters/operate-exporter/src/main/java/io/camunda/zeebe/operate/exporter/handlers/DecisionRequirementsHandler.java#L34 the intent accepted to handle record was `ProcessIntent.CREATED`, but I thought this was by mistake?

Reply:
According to the [old importer](https://github.com/camunda/camunda/blob/49b86c467022dceab72c0ad592272750779aa9dc/operate/importer-8_7/src/main/java/io/camunda/operate/zeebeimport/processors/DecisionRequirementsZeebeRecordProcessor.java#L30) it seems that the ProcessIntent was correct. 

But @sdorokhova has more context in this.

Reply:
Good catch, @mustafadagher ! I think it was a copy-paste error, that worked just because they have the same names.